exonum/src/runtime/types.rs

```diff
@@ -167,9 +167,9 @@ impl ArtifactId {
 
     /// Check that the artifact name contains only allowed characters and is not empty.
     fn is_valid_name(name: impl AsRef<[u8]>) -> bool {
-        // Extended version of `exonum_merkledb::is_valid_name` that also allows ':`.
+        // Extended version of `exonum_merkledb::is_valid_name` that also allows '.' and ':'.
```

Comment:
Is dot allowed for artifact names?

Reply:
Based on task description for artifact names yes, for services not.